docker-compose.dev.yml

```diff
@@ -7,6 +7,7 @@ services:
     build:
       context: .
       target: base
+    init: true
```

Comment:
(not related) This is just to make the container stop faster. Without it it looks like signals (SIGINT, etc..) are not forwarded to `tail` thus making docker wait for a timeout before killing the container.